vignettes/desctable.Rmd

```diff
@@ -288,6 +283,8 @@ But what if you want to pick a specific test for a specific variable, or change
   * an automatic function to select appropriate statistical test functions
   * or a named list of statistical test functions
 
+The named list can contain named functions such as `t.test` prefixed with *~* but also pipelines using *pipe* character from **magrittr** and **(.)** argument. Please note, the necessity to use **~** before any function, due to use of **rlang** `as_function`  function.
```

Comment:
Don't mention pipelines, always use "tidy" formulas. Don't mention rlang::as_function, but "tidyverse formula" (look for the appropriate name in tidy manifesto or purrr docs and vignettes)